tests/Feature/Http/Controllers/CampaignControllerTest.php

```diff
@@ -76,9 +77,37 @@ public function store_saves_and_redirects()
         $adventure = Adventure::factory()->create();
         $title = $this->faker->sentence(4);
 
+        $response = $this->post(route('campaign.store'), [
+            'adventure_id' => $adventure->id,
+            'title' => $title
+        ]);
+
+        $campaigns = Campaign::query()
+            ->where('adventure_id', $adventure->id)
+            ->where('title', $title)
+            ->get();
+        $this->assertCount(1, $campaigns);
+        $campaign = $campaigns->first();
+
+        $response->assertRedirect(route('campaign.index'));
+        $response->assertSessionHas('campaign.id', $campaign->id);
+        $this->assertDatabaseCount('campaign_user', 1);
+        $this->assertDatabaseHas('campaign_user', ['user_id' =>$this->user->id, 'campaign_id' =>$campaign->id, 'is_dm' =>true]);
+    }
+
+    /**
+     * @test
+     */
+    public function store_saves_character_and_redirects()
```

Comment:
shouldnt this be `store_saves_campaign_...` not `store_saves_character_...` ?

Reply:
```suggestion
    public function store_saves_campaign_and_redirects()
```